src/main/kotlin/nexstep/mission/calculator/Calculator.kt

```diff
@@ -0,0 +1,18 @@
+package nexstep.mission.calculator
+
+fun String.isInteger() = this.toIntOrNull()?.let { true } ?: false
```

Comment:
```suggestion
fun String.isInteger() = this.toIntOrNull() != null
```

null 비교만 해도 괜찮지 않을까요~?